### ISSUE TITLE
Parse.User.save() does not work in parse@3.5.0

### ISSUE THREAD

**Issue by dipendras** [parse-community/Parse-SDK-JS#1593]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the version *4.5.0* of [Parse Server](https://github.com/parse-community/parse-server/releases) and the latest version of [Parse JS SDK](https://github.com/parse-community/Parse-SDK-JS/releases). 

### Issue Description
Parse.User.save() does not work in parse@3.5.0

### Steps to reproduce
let user = new Parse.User();
user.set("username", "xx");
user.set("email", "xx@xx.xx");
user.set("password", "xx");
saveduser = await user.save();

### Actual Outcome
It is saved/updated in the local session but not in the backend/server.

### Expected Outcome
Update the data in the backend/server.

###  Environment

Server
- Parse Server version: `4.5.0`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `Back4app`

Client
- Parse JS SDK version: `3.5.0`

**Comment by mtrezza** [comment]
> I can reproduce the issue with the version 4.5.0 of [Parse Server](https://github.com/parse-community/parse-server/releases) a

Can you reproduce this issue with the latest alpha version of Parse Server 5?

**Comment by mtrezza** [comment]
> I can reproduce the issue with the version 4.5.0 of [Parse Server](https://github.com/parse-community/parse-server/releases) 

Can you reproduce this issue with the latest alpha version of Parse Server 5?

**Comment by marcderhammer** [comment]
I believe this also applies to 
`Parse.User.signUp`

When calling this, it is not sending any network request.
Downgrading to 3.4.4 resolves it for me.

`logIn` is fine

**Comment by mtrezza** [comment]
Which Parse Sever version did you use?

**Comment by marcderhammer** [comment]
I was on server 5.2.4, but bumping up to 5.3.0 did not resolve it.

```
const signUp = async (username, password) => {
    try {
        const user = new Parse.User();
        user.set("username", username);
        user.set("email", username);
        user.set("password", password);
        await user.signUp();
    } catch (error) {
        alert(error.message);
    }
}
```

the above code works well on JS SDK 3.4.4 but not 3.5.0. 

Watching the network tab on developer tools shows no request when calling `signUp`

maybe worth noting I'm pulling it from this [CDN](https://unpkg.com/parse@3.5.0/dist/parse.min.js)

**Comment by mtrezza** [comment]
The CI for release branch indeed shows that there are some tests failing:
https://github.com/parse-community/Parse-SDK-JS/actions/runs/3373095961/jobs/5649179081

Looking at the badges, the alpha and beta branches build successfully, only the release branch is failing:
<img width="280" alt="image" src="https://user-images.githubusercontent.com/5673677/200080227-df918702-3183-41ec-8908-2e767a1bf5c5.png">

Could you try with the latest alpha and beta version of the Parse JS SDK?

**Comment by aarondill** [comment]
Upon running my tests using Query.subscribe() and Query.first(), I received the same errors in the JS SDK versions v3.5.1-beta-1, v3.5.1-beta-1, and production v3.5.0. admittedly, those tests didn't include the use of User.save, but it still seems relevant.

**Comment by mtrezza** [comment]
@MarcDerhammer Is there no log output at all when this fails? Could you try this in a local environment and run the integration test for signup? I assume there is one.

**Comment by aarondill** [comment]
upon further investigation, my previous comment seems irrelevant, due to the different causes of the issues.

I will leave this issue with a comment pointing out the only changes between the User.save() functions shown in v3.4.4 and v3.5.0:

In v3.4.4:
```js
return _regenerator.default.wrap(function (_context2) {...})
```

in v3.5.0:
```js
return _regeneratorRuntime().wrap(function (_context2) {...})
```

the contents of the two functions are identical. the only difference is the call to _\_regenerator.default.wrap_ versus _\_regeneratorRuntime().wrap_.

As I don't understand what the code does in each case, I will leave this here for more knowledgeable people. I wish you luck.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1-alpha.2](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-alpha.2)

**Comment by mtrezza** [comment]
Closing via https://github.com/parse-community/Parse-SDK-JS/pull/1600

**Comment by mtrezza** [comment]
This should be fixed with https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-alpha.2, could you try it out and let us know?

**Comment by marcderhammer** [comment]
Just confirmed using `3.5.1-alpha.2` resolved the `Parse.User.signUp` problem I was having.

Thank you very much!

**Comment by dipendras** [comment]
I confirm that the issue is fixed with parse@3.5.1-alpha.2.

**Comment by livingstonef** [comment]
I am not sure what i am doing wrong here, but for me `Parse.User.signUp` in parse@3.5.1-alpha.2 is still not sending any request to the server. `Parse.User.logIn` works well though.

This is using Parse server version 6.0.0-alpha.9

**Comment by mtrezza** [comment]
Since the original issue has been solved for others, it may be a different issue. Did you ensure the dependency installed is `parse@3.5.1-alpha.2`, not just in package.json?

**Comment by livingstonef** [comment]
Thank you for the reply. I am confident I was using `3.5.1-alpha.2 `

Anyways. The issue is now gone when using `4.0.0.alpha.3`. Thanks again.

**Comment by mtrezza** [comment]
Thanks for reporting back.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1-beta.2](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1)
